pxtblocks/fields/field_sprite.ts

```diff
@@ -99,6 +99,9 @@ namespace pxtblockly {
             });
 
             this.editor.setActiveColor(this.params.initColor, true);
+            if (!this.params.sizes.some(s => s[0] == this.state.width && s[1] == this.state.height)) {
```

Comment:
Not sure of your team's style guidelines, but we usually enforce strict equality checks unless we explicitly need them to be loose.

Reply:
I haven't seen any guidelines for this in particular, but it is always good practice to use strong checks where applicable. Thanks for the note!